index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html>
+   <head>
+<script src="script.js"></script>       
+
+   <body>
+       <h1>Web Calculator</h1>
+       <form>
+       <input type="text" id="value1" value=""/>
+       <select id="operator">
+           <option value="+">+</option>
+           <option value="-">-</option>
+           <option value="*">*</option>
+           <option value="/">/</option>
+       </select>
+       <input type="text" id="value2" value=""/>
+       <input type="button" onclick="calculate()" style="background-color: darkorchid ;" onclick="sum();" value=" Submit"/>
```

Comment:
You have a random space before Submit

You should use a stylesheet for CSS. Don't put it "inline", is what this is called. But you get extra points for just having it at all.